src/qibolab/instruments/rfsoc.py

```diff
@@ -180,6 +180,8 @@ def _open_connection(host: str, port: int, server_commands: dict):
                     break
                 received.extend(tmp)
         results = pickle.loads(received)
+        if isinstance(results, Exception):
+            raise RuntimeError(f"An error occured on the server side: {results}")
```

Comment:
@rodolfocarobene thanks for implementing this. Just a curiosity, if you do
```py
raise results
```
will it work? That's so that the error raised on the client side is of the same type with the error in the server, instead of always being `RuntimeError`

Reply:
Yes it works, I will commit it and push it somewhere else, thanks!